src/commonMain/kotlin/exchange.dydx.abacus/state/manager/V4StateManagerAdaptor.kt

```diff
@@ -1064,7 +1064,7 @@ class V4StateManagerAdaptor(
                             )
                         }
                     }
-                    send(error, callback, it)
```

Comment:
nit: for lambda blocks longer than one line, we should used named params instead of `it` (as an aside, we should really get a linter/static-analysis tool setup, there's a simple lint rule for this)

Reply:
ooh will fix!